src/sentry/utils/sdk.py

```diff
@@ -120,6 +120,7 @@
 
 UNSAFE_TAG = "_unsafe"
 EXPERIMENT_TAG = "_experimental_event"
+HYBRID_CLOUD_TEST_BED_TARGET_TAG = "_hybrid_cloud_test_bed"
```

Comment:
I don't understand why we are `_` prefixing tag values for our own application. Its not like we need to worry about conflicts with other parts of our application.

Reply:
I also don't understand, but I used the rule of "follow patterns established" when in doubt 🤷 

Possibly, the idea was also to have these tags sort differently so as not to mix it with other categorically different tags???

Reply:
Could be a sorting thing. I'm not sure why any of these tags have `_` prefixes. It isn't that important though.